pkg/chunk/gcp/bigtable_index_client.go

```diff
@@ -50,16 +53,18 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.StringVar(&cfg.Instance, "bigtable.instance", "", "Bigtable instance ID.")
 	f.BoolVar(&cfg.TableCacheEnabled, "bigtable.table-cache.enabled", true, "If enabled, once a tables info is fetched, it is cached.")
 	f.DurationVar(&cfg.TableCacheExpiration, "bigtable.table-cache.expiration", 30*time.Minute, "Duration to cache tables before checking again.")
-
+	f.IntVar(&cfg.WriteLimit, "bigtable.write-rate-limit", 0, "BigTable rate limiter for write operations, with 0 is disable.")
```

Comment:
`0 is disabled` rather than `with 0 is disabled`? On this line and the line after?